search/lib/api/wfs.js

```diff
@@ -32,8 +33,20 @@ async function getGranules (request, response) {
   const conceptId = request.params.collectionId;
   const params = Object.assign({ collection_concept_id: conceptId }, cmr.convertParams(cmr.WFS_PARAMS_CONVERSION_MAP, request.query));
   const granules = await cmr.findGranules(params);
+
+  const currPage = parseInt(extractParam(event.queryStringParameters, 'page_num', '1'), 10);
+  const nextPage = currPage + 1;
+  const newParams = { ...event.queryStringParameters } || {};
+  newParams.page_num = nextPage;
+  const nextResultsLink = generateAppUrlWithoutRelativeRoot(event, event.path, newParams);
+
   const granulesResponse = {
-    features: granules.map(gran => convert.cmrGranToFeatureGeoJSON(event, gran))
+    type: 'FeatureCollection',
+    features: granules.map(gran => convert.cmrGranToFeatureGeoJSON(event, gran)),
+    links: {
```

Comment:
I think `links` is supposed to be an array according the spec here https://github.com/radiantearth/stac-spec/blob/v0.8.1/item-spec/itemcollection-spec.md

Reply:
You're right, I've fixed this in the latest commit.